addons/web/static/src/search/search_arch_parser.js

```diff
@@ -160,7 +172,9 @@ export class SearchArchParser extends XMLParser {
                         preSearchItem.defaultIntervalId = defaultInterval || DEFAULT_INTERVAL;
                     }
                 }
-            } catch (e) {}
+            } catch (e) {
```

Comment:
what could crash here? We should only put in the try {} what can throw the error we want to catch